docs/src/pages/component-api/Tabs/Tabs.md

```diff
@@ -8,17 +8,17 @@ Props
 
 | Name | Type | Default | Description |
 |:-----|:-----|:--------|:------------|
-| buttonClassName | string | | The CSS class name of the manual scroll buttons. |
+| buttonClassName | string |  | The CSS class name of the scroll button elements. |
 | centered | bool | false | If `true`, the tabs will be centered. This property is intended for large views. |
 | children | node |  | The content of the component. |
 | className | string |  | The CSS class name of the root element. |
 | fullWidth | bool | false | If `true`, the tabs will grow to use all the available space. This property is intended for small views. |
-| index | number |  | The index of the currently selected `Tab`. |
+| index | number |  | The index of the currently selected `BottomNavigation`. |
```

Comment:
@oliviertassinari Out of curiosity, why was this line changed back to the original form?  I had assumed the usage of `BottomNavigation` instead of `Tab` was an oversight, but maybe I am mistaken.